frontend/src/screens/channels/Channels.tsx

```diff
@@ -584,7 +606,19 @@ export default function Channels() {
         <Card className="flex flex-col">
           <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
             <CardTitle className="text-sm font-medium">
-              Receiving Capacity
+              <TooltipProvider>
+                <Tooltip>
+                  <TooltipTrigger>
+                    <div className="flex flex-row gap-2 items-center">
+                      Receiving Capacity
+                      <InfoIcon className="h-4 w-4 shrink-0 text-muted-foreground" />
+                    </div>
+                  </TooltipTrigger>
+                  <TooltipContent>
+                    Shows how much bitcoin you can currently receive
```

Comment:
```suggestion
                    Your receiving balance is the funds owned by your channel partner, which will be moved to your side when you receive lightning payments.
```

Reply:
I took these tooltip descriptions from [here](https://www.figma.com/design/2IyilDewDrdNP3G1NixPYd/Alby-Hub?node-id=4200-59372&t=ToHa0bXLMk56hWbk-1), we don't need one-liners?

Reply:
How do the tooltips look now? I think there were some issues in the original copy, which I tried to resolve with my suggestions